x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/transport/filter/IPFilterTests.java

```diff
@@ -67,18 +76,19 @@ public void init() {
         auditTrailService = new AuditTrailService(Collections.singletonList(auditTrail), licenseState);
         clusterSettings = new ClusterSettings(
             Settings.EMPTY,
-            new HashSet<>(
-                Arrays.asList(
-                    IPFilter.HTTP_FILTER_ALLOW_SETTING,
-                    IPFilter.HTTP_FILTER_DENY_SETTING,
-                    IPFilter.IP_FILTER_ENABLED_HTTP_SETTING,
-                    IPFilter.IP_FILTER_ENABLED_SETTING,
-                    IPFilter.TRANSPORT_FILTER_ALLOW_SETTING,
-                    IPFilter.TRANSPORT_FILTER_DENY_SETTING,
-                    IPFilter.PROFILE_FILTER_ALLOW_SETTING,
-                    IPFilter.PROFILE_FILTER_DENY_SETTING
-                )
-            )
+            Stream.of(
+                IPFilter.HTTP_FILTER_ALLOW_SETTING,
+                IPFilter.HTTP_FILTER_DENY_SETTING,
+                IPFilter.IP_FILTER_ENABLED_HTTP_SETTING,
+                IPFilter.IP_FILTER_ENABLED_SETTING,
+                IPFilter.TRANSPORT_FILTER_ALLOW_SETTING,
+                IPFilter.TRANSPORT_FILTER_DENY_SETTING,
+                TcpTransport.isUntrustedRemoteClusterEnabled() ? IPFilter.REMOTE_CLUSTER_FILTER_ALLOW_SETTING : null,
+                TcpTransport.isUntrustedRemoteClusterEnabled() ? IPFilter.REMOTE_CLUSTER_FILTER_DENY_SETTING : null,
+                IPFilter.PROFILE_FILTER_ALLOW_SETTING,
+                IPFilter.PROFILE_FILTER_DENY_SETTING,
+                TcpTransport.isUntrustedRemoteClusterEnabled() ? RemoteClusterPortSettings.REMOTE_CLUSTER_PORT_ENABLED : null
```

Comment:
That's the `remote_cluster.enabled` setting, and we need it in these tests because we don't set up the `_remote_cluster` profile at all (including the IP filters) if that setting isn't set. `remote_cluster` is explicitly different from other profiles in that we said it should be possible to configure without using the `transport.profiles.*` settings.